kvirt/providers/ibm/__init__.py

```diff
@@ -190,8 +190,8 @@ def create(self, name, virttype=None, profile='', flavor=None, plan='kvirt',
             return {'result': 'failure', 'reason': 'Unable to check provisioned images. %s' % exc}
 
         volume_attachments = []
-        for index, disk in enumerate(disks[1:]):
```

Comment:
i believe the primary disk gets created independently nor can it be resized so we dont want to handle it ourselves

Reply:
The primary disk (or boot volume) is created automatically by IBM cloud and you cant do anything from here. If you consider the rest of the disks to be user-defined, then the old loop was skipping the first one.
If you happen to have only one user defined disk it is not going to be created.
Maybe we should just remove the default value?

Reply:
my point is that when you say my vm will have two disks, it means the OS one and a secondary one,
without the slice, i d get 3 disks

Reply:
You would only get three disks if you define two as a user, but that means there should be 3 according to how ibm cloud works. The first one comes from the image itself, but the rest are an add-on you define. If you skip the first one you are creating less disks than desired.
For example if I define a 300G additional disk, and just that, it would be skipped and the VM would only have the boot disk. You would need to define 2 disks and the first one is always silently ignored.

Reply:
while i understand your point of view, i think it's more misleading for the end user.
if you say your vm will have 2 disks, and in general you mean one of those disks to be your OS one that's what should happen and get reflected.
ibmcloud has a limitation on the first disk not beeing resized, but that could change in the future. Furthermore, kubevirt has a similar behaviour.

Reply:
I will close the PR because of uniformity, just checked how AWS and GCP work too. Thanks!